docs/8-veba-troubleshooting.md

```diff
@@ -147,9 +158,11 @@ Here is the command output:
 ```
 We now see that the Event Router came online, connected to vCenter, and successfully received an event.
 
+<BR>
```

Comment:
Not needed, see comment above.

Reply:
I can put as many new lines as I like, it doesn't actually create any space. 

This is with 4 new lines
![image](https://user-images.githubusercontent.com/29929717/77233220-24713a00-6b74-11ea-8d27-429ddbc8560a.png)

and this is with a BR tag
![image](https://user-images.githubusercontent.com/29929717/77233234-37840a00-6b74-11ea-8991-585e6c5fa965.png)

Reply:
I see what you mean. Don't worry about the spaces. Just go with the defaults git uses for markdown rendering. If you want a space between lines just add an empty line between them. If you want to force line-wrap add two spaces after the sentence.